frontend/src/components/Header.jsx

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import { Cookies } from "react-cookie";
```

Comment:
so I'm understanding, you are making and storing the cookie in the header? Or is it also made in sign up? I just want to see how I will move it into reviews, probably by lifting state.

Reply:
Right now, cookies are set in SignIn.jsx, and then you can access them from wherever. If you look at "profile.jsx" you can see how you create a variable "const cookie = new Cookie()" and then can access the cookies with "cookie.get("token")" or "cookie.get("userId")".

Reply:
I'll look at the bug to see if I can fix it. Thanks for the heads up.

Reply:
I got it working on my end, let me know if it works for you.